.github/workflows/backend.yml

```diff
@@ -111,15 +111,17 @@ jobs:
         run: psalm --output-format=github --php-version=${{ matrix.php }} --report=sarif/psalm.sarif --report-show-info=false
 
       - name: Upload coverage results to Codecov
+        env:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          PHP: ${{ matrix.php }}
+        if: env.token != ''
```

Comment:
I know it's extreme but what if the forked repo has the secret value `CODECOV_TOKEN`?

If the above situation is possible, the repository name can be checked. 

````yaml
if: ${{ github.repository == 'getkirby/kirby' }}
````

TBH I'm not sure. I'm just thinking out loud.

Reply:
I tried that as well, but that would break for the PR run. In this case we *do* want the Codecov reporting. My if logic to fix this required three lines, so this is much simpler.

Also I think it's not an issue. If the secret value is set, then Codecov will work as expected.

Reply:
That's right, we want it to run, not throw an error and fail after all. 👍